src/renderers/WebGLRenderer.js

```diff
@@ -1166,6 +1168,30 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	// Rendering
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, forceClear ) {
+
+		scene.overrideMaterial = overrideMaterial;
+		this.render( scene, camera, renderTarget, forceClear );
+		scene.overrideMaterial = null;
+
+	};
+
+	this.renderPass = function ( passMaterial, renderTarget, forceClear ) {
+
+		if( passScene === null ) {
+			passCamera = new THREE.OrthographicCamera( -1, 1, 1, -1, 0, 1 );
+			passQuad = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2, 2 ), null );
+			passScene = new THREE.Scene();
+			passScene.add( passQuad );
+		}
+
+		passQuad.material = passMaterial;
+		this.render( passScene, passCamera, renderTarget, forceClear );
+		passQuad.material = null;
```

Comment:
Hmm... The last three lines are the same as

    this.renderOverride( passMaterial, passScene, passCamera, renderTarget, forceClear );